spec/features/budgets/votes_spec.rb

```diff
@@ -1,9 +1,9 @@
 require "rails_helper"
 
 feature "Votes" do
+  let(:manuela) { create(:user, verified_at: Time.current) }
 
-  feature "Investments" do
-    let(:manuela) { create(:user, verified_at: Time.current) }
+  feature "Investments - Knapsack" do
```

Comment:
How about using a context here too? something like this:

```
context "Knapsack"
  feature "Investments" do
  end
end

context "Approval Voting"
  feature "Investments" do
  end
end
```

Note that the new Rails 5.1 syntax for `feature` is now `describe`. Feel free to use the new syntax 🎉
https://github.com/consul/consul/pull/3562/commits/307cf248468102b9afef0efab7fd8d387601d88a

Reply:
I will try it. I thought Capybara used `feature` and `scenario`.